From 0c944e414671384ed82bbba6ba673768a6228943 Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:02:56 -0400
Subject: [usb] wusbcore: fix build warning on 64-bit builds

Message-id: <1410887463-20674-105-git-send-email-dzickus@redhat.com>
Patchwork-id: 95120
O-Subject: [RHEL7 PATCH 104/591] usb: wusbcore: fix build warning on 64-bit builds
Bugzilla: 1110939
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=1110939

commit cccd3a258eef7a8451e25b7bf55503973de4e9f5
Author: Thomas Pugliese <thomas.pugliese@gmail.com>
Date:   Mon Sep 30 22:48:46 2013 -0500

    usb: wusbcore: fix build warning on 64-bit builds

    Fix a build warning found by the kbuild test robot in the most recent
    wusbcore patches.

    Signed-off-by: Thomas Pugliese <thomas.pugliese@gmail.com>
    Reported-by: kbuild test robot <fengguang.wu@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/wusbcore/wa-xfer.c b/drivers/usb/wusbcore/wa-xfer.c
index d260836..9dabd89 100644
--- a/drivers/usb/wusbcore/wa-xfer.c
+++ b/drivers/usb/wusbcore/wa-xfer.c
@@ -301,7 +301,7 @@ static unsigned __wa_xfer_is_done(struct wa_xfer *xfer)
    break;
   case WA_SEG_ERROR:
    xfer->result = seg->result;
-   dev_dbg(dev, "xfer %p ID %08X#%u: ERROR result %zu(0x%08X)\n",
+   dev_dbg(dev, "xfer %p ID %08X#%u: ERROR result %zu(0x%08zX)\n",
     xfer, wa_xfer_id(xfer), seg->index, seg->result,
     seg->result);
    goto out;
-- 
1.7.1